### ISSUE TITLE
Clean up current list

### ISSUE THREAD

**Issue by fmerian** [agamm/awesome-developer-first#167]
We just hit 1,000 GitHub stars!

To keep the list awesome, we need to go over all the links and check if everything is still up. If not, move links to [OUTCAST.md](/agamm/awesome-developer-first/blob/main/OUTCAST.md).

Thanks!

**Comment by agamm** [comment]
Yes. Perfect first issue. I'll try to work category by category over time too.
